README.md

```diff
@@ -15,6 +15,51 @@ Machine learning potentials can be installed directly from [scripts/](scripts).
 - Units are: distance (Å), energy (eV), force (eV Å-1), time (fs)
 - Training using molecular mechanics (MM) is not supported as we've not found it to be efficient
 
+
+## Using with OpenMM
+
+The OpenMM backend only works with MACE at the moment
+
+First install requirements from conda-forge
+
+```
+mamba install -c conda-forge autode=1.1* python=3.9 openmm-torch=1.1 pytorch=2.0
+```
+
+Then install this version of ASE:
+```
+pip install git+https://github.com/rosswhitfield/ase.git
+```
+
+Then install MACE:
+```
+pip install git+https://github.com/ACEsuit/mace.git
+```
+
+Then install this specific fork of OpenMM-ML:
+```
+pip install git+https://github.com/sef43/openmm-ml@mace
```

Comment:
Should check if this is merged, and if not contact openmm team to merge. (not blocking for this PR)

Reply:
There is an open PR https://github.com/openmm/openmm-ml/pull/61
We should link to it in the README as well. I guess for now installing for the branch is fine, but we should push them a bit to merge it soon.